app/scripts/controllers/transactions/tx-state-manager.test.js

```diff
@@ -648,152 +824,44 @@ describe('TransactionStateManager', function () {
 
   describe('#getTx', function () {
```

Comment:
Forgot to update this description:
```suggestion
  describe('#getTransaction', function () {
```

Reply:
Thanks!